RepairsListener/Infrastructure/AddRepairsContractsToNewAssetObject.cs

```diff
@@ -0,0 +1,16 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace RepairsListener.Infrastructure
+{
+    // The same class is also defined in Asset API to ensure the information sent and received is in the same format
+    public class AddRepairsContractsToNewAssetObject
+    {
+        public Guid EntityId { get; set; }
+        public string PropRef { get; set; }
+        public bool AddRepairsContracts { get; set; }
```

Comment:
Should we default this to true if no value is provided? (just add `= true; `)

I'm feeling that we should err on the side of caution for malformed messages (which is more common when not using a shared library). Caution, in this instance, is creating the contracts

Reply:
So do you think we should still include this AddRepairsContracts bool property? Given our discussion in this [PR](https://github.com/LBHackney-IT/asset-information-api/pull/124) I thought we could completely get rid of it, and only send the AddContracts event, in those cases when we want to add contracts for (new) assets, which makes this property no longer necessary, no?

Reply:
Removed AddRepairsContracts property from object (following its removal from Asset API)